src/test/java/guitests/guihandles/PersonListPanelHandle.java

```diff
@@ -76,10 +76,13 @@ public ReadOnlyViewablePerson getSelectedPerson() {
     }
 
     /**
-     * Clicks on the middle of the Listview.
+     * Clicks on the middle of the Listview if the listview is not in focus
      * In order for headfull testing to work in travis ci, listview needs to be clicked before firing hot keys.
      */
     public void clickOnListView() {
```

Comment:
If you go ahead with this, rename method to 'ensureListViewHasFocus' (because clicking is no longer guaranteed).